.github/workflows/deploy_release.yml

```
@@ -34,3 +34,34 @@ jobs:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
           packages_dir: pyodide-build/dist/
+
+  deploy-pyodide-py-pypi:
+    runs-on: ubuntu-latest
+    environment: PyPi
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```

Probably `deploy-pyodide-build-pypi` job should also be fixed